tensorrt_llm/_torch/models/modeling_nemotron_h.py

```diff
@@ -1,19 +1,18 @@
-from typing import Dict, Optional, Tuple
+from typing import Dict, Optional
 
 import torch
 from torch import nn
 from torch.nn import functional as F
+from transformer_engine.pytorch import RMSNorm
```

Comment:
I think it should be 

```python
from ..modules.mamba.layernorm_gated import RMSNorm
```

Reply:
The transformer engine `RMSNorm` was the one used for the `NemotronHLayer` in Megatron and the `RMSNorm` from `layernorm_gated` is only used within the `MambaMixer` module.

Reply:
`transformer_engine` is a new dependency. It's installed in NGC PyTorch container by default. However, bare metal Ubuntu container does not contain this library. cc @chzblych 

Can we port it via source codes? Or use another `RMSNorm` implementation?

Reply:
I used try/except to do the import and later assert that `RMSNorm is not None` with a message saying model needs transformer engine and you should pip install it with this.